template/src/components/views/stores/StoresDetail.js

```diff
@@ -42,6 +45,21 @@ export default function StoresDetail() {
   const source = useSelector((state) => selectSourceById(state, 'storesSource'));
   const navigate = useNavigate();
 
+  const xAxisLabels = [
```

Comment:
Move this down. The first element after the imports should be the Component function

Reply:
Better move this to common.js and rename common.js to constants.js